### PR TITLE
Git fork vs Git clone

### PR DESCRIPTION
**Pull request by SukritaEarn**

### CODE REVIEW

File: git-fork-vs-git-clone/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    
+    e. Create a local clone of your fork
+    
+    f. Directly commit to the original repository
+    
+    g. Pull changes from the original repository
+
+**Answer**
+
+Answer 1 :
```
Comment (jbrucker) [inline git-fork-vs-git-clone/README.md]: Punctuation spacing error (mentioned several times in class that you should
not do this!)

File: git-fork-vs-git-clone/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+* Forking
+
+The forked repository is mostly static. It exists in order to allow you to publish work for code review purposes. You can't do active development in your forked repository because it doesn't exist on your computer, it exists on GitHub's server in the cloud. The changes must then be pulled into the source repository by the project maintainer.
```
Comment (jbrucker) [inline git-fork-vs-git-clone/README.md]: You can do work in a forked repository, including creating new branches.
Clone it to your computer, just like any repo, do work, and push back to your forked repo.

The differences I see are Issues and how Pull Requests work.

**Comment by SukritaEarn** [comment]
I fixed them. Please review again.

**Comment by jbrucker** [comment]
Looks better now.

**Pull request closed without merging**